Shopify/cli
# Declarative webhooks relative paths

### WHY are these changes introduced?

Closes: https://github.com/Shopify/develop-app-management/issues/1599

Adds relative URIs to your app's URL to declarative webhooks

### WHAT is this pull request doing?

- exposes the `fullAppConfiguration` to various deploy and link locations in order to transform webhook subscriptions by checking the `application_url`

### How to test your changes?

- `deploy` and `link` with relative subscription URIs, check the config to make sure it inherits the app url
